src/configs/gemm-config.c

```diff
@@ -1419,6 +1422,17 @@ static void init_f32_qc8w_gemm_config(void) {
   #endif
 }
 
+static void init_qd8_f16_qb4w_gemm_config(void) {
```

Comment:
Not sure if we want to include this in this PR (both scalar and / or benchmarking) because this is mainly related to the operator level support which we are not doing at the moment. Just kernel and tests. And benchmarking as a second step. So let's drop this for now? It was needed for internal because of e2e support but not for Google.

Reply:
I can pull this out.